src/utilities.js

```diff
@@ -122,6 +122,16 @@ export function getResults(searchType, obj) {
 
   return request(options)
   .then((results) => {
+
+    // If this fails, you've hit the rate limit or Google has changed something
+    try {
+      JSON.parse(results.slice(4));
+    } catch (e) {
+      // Throw the JSON error - This should perhaps throw an object with the
```

Comment:
Yea that's a good idea.  Let's do that except let's put the `try catch` around line 135 rather than having a separate `JSON.parse`